x/tally/keeper/gas_meter.go

```diff
@@ -130,17 +130,20 @@ func MeterExecutorGasFallback(req types.Request, gasCostFallback uint64, gasMete
 
 // MeterExecutorGasUniform computes and records the gas consumption of executors
 // when their gas reports are uniformly at "gasReport".
```

Comment:
Maybe good to describe the outlier behaviour in the comment here and lines 144-145

Reply:
ah yes, fixed!